content/docs/digging_deeper/transmit.md

```diff
@@ -0,0 +1,373 @@
+# Transmit
+
+Transmit is a native Server-Sent-Event (SSE) module built for AdonisJS. It is a simple and efficient way to send real-time updates to the client, such as notifications, live chat messages, or any other type of real-time data.
+
+:::note
+The data transmission occurs only from server to client, not the other way around. 
```

Comment:
You could add that to do from client to server, dev can use a form or a fetch. It could sound obvious but after reading many articles about SSE that's one of the "issue"  raised. (the unilateral communication)